projects/the-application/src/assets/i18n/main/hi.json

```diff
@@ -1,17 +1,17 @@
 {
-  "ADD_TO_HOME_SCREEN": "ADD_TO_HOME_SCREEN",
-  "CLOSE": "CLOSE",
-  "DESCRIPTION": "DESCRIPTION",
-  "FULL_STOP": "FULL_STOP",
-  "GAME": "GAME",
-  "HIGH_SCORES": "HIGH_SCORES",
-  "INSTALL": "INSTALL",
-  "INSTALL_MEMORY_GAME": "INSTALL_MEMORY_GAME",
-  "PLEASE_RELOAD": "PLEASE_RELOAD",
-  "PLEASE_FINISH_OR_RESET_GAME": "PLEASE_FINISH_OR_RESET_GAME",
-  "RECENT_SCORES": "RECENT_SCORES",
-  "REFRESH": "REFRESH",
-  "SETTINGS": "SETTINGS",
-  "TITLE": "TITLE",
-  "UPDATE_AVAILABLE": "UPDATE_AVAILABLE"
+  "ADD_TO_HOME_SCREEN": "होम स्क्रीन में शामिल करें",
+  "CLOSE": "बंद करे",
+  "DESCRIPTION": "मनुष्यों, कुत्तों और बिल्लियों के लिए मेमोरी कार्ड गेम",
+  "FULL_STOP": "|",
+  "GAME": "खेल",
+  "HIGH_SCORES": "उच्च स्कोर",
+  "INSTALL": "इंस्टॉल",
+  "INSTALL_MEMORY_GAME": "मेमोरी गेम इंस्टॉल करें",
+  "PLEASE_RELOAD": "कृपया पुनः लोड करें",
+  "PLEASE_FINISH_OR_RESET_GAME": "कृपया गेम समाप्त करें या रीसेट करें",
+  "RECENT_SCORES": "हाल ही में स्कोर",
+  "REFRESH": "रीफ़्रेश करें",
+  "SETTINGS": "सेटिङ्",
```

Comment:
Can you add TAP and THEN?

See https://github.com/tedaky/memory-game/pull/16/files#diff-47c31de70082a49e837b88ef47ba61dc365e3d92008c6d09cbacc1a0edbfb98a